src/device/io/map.c

```diff
@@ -46,6 +46,9 @@ static inline void invoke_callback(io_callback_t c, paddr_t offset, int len, boo
 word_t map_read(paddr_t addr, int len, IOMap *map) {
   assert(len >= 1 && len <= 8);
   check_bound(map, addr);
+  if((MMIO_READ & map->skip_ref_flag) != 0) {
```

Comment:
replace 0 with skip_ref

Reply:
I think 0 is correct. flag can include multiple flags. right?

Reply:
> I think 0 is correct. flag can include multiple flags. right?

yes, i'll replace 0 with SKIP_FREE